### PR TITLE
sync up easyconfig for IPython 5.7.0 w/ foss/2018a & Python 2.7.14 wi…

### PR DESCRIPTION
**Pull request by boegel**
…th equivalent using intel/2018a toolchain

see discussion in #6776

I would be nice to get this merged before #6776 (since reviewing #6676 then becomes trivial, there's literally no other difference than the toolchain name anymore...).

@smoors Please review (since this is tweaking the easyconfig you contributed in #6415, let me know if you have any concerns.

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2689.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c1823569af61ad47e2a1584fa052a828 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2104.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/81f4675332aaaaafa0f2364bd93ca0c7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in this PR)
node3147.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/0179a1de45d04de5039e875bcbf84fe2 for a full test report.

File: easybuild/easyconfigs/i/IPython/IPython-5.7.0-foss-2018a-Python-2.7.14.eb [hunk 1]
```diff
@@ -16,23 +16,25 @@ toolchain = {'name': 'foss', 'version': '2018a'}
...
+        'source_urls': ['https://pypi.python.org/packages/source/p/pyzmq/'],
+        'checksums': ['0145ae59139b41f65e047a3a9ed11bbc36e37d5e96c64382fcdff911c4d8c3f0'],
+        'modulename': 'zmq',
     }),
     ('nbformat', '4.4.0', {
         'source_urls': ['https://pypi.python.org/packages/source/n/nbformat/'],
         'checksums': ['f7494ef0df60766b7cabe0a3651556345a963b74dbc16bc7c18479041170d402'],
     }),
     ('Pygments', '2.2.0', {
-        'modulename': 'pygments',
```
Comment (smoors) [inline easybuild/easyconfigs/i/IPython/IPython-5.7.0-foss-2018a-Python-2.7.14.eb]: question: why did you remove the modulename here? I thought it was needed for the sanity check `python -c 'import pygments'`, but apparently that is not the case since you installed successfully. Does EB automatically convert to lowercase?

Comment (boegel) [reply easybuild/easyconfigs/i/IPython/IPython-5.7.0-foss-2018a-Python-2.7.14.eb]: Indeed, the default module name is the software name but lowercase (which is the most common pattern for Python packages), so `Pygments` -> `import pygments` is done automatically.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.9, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/d3952afe7312eab46ab929bc5e6fa846 for a full test report.
